intera_core_msgs/msg/InteractionControlCommand.msg

```diff
@@ -57,3 +57,10 @@ uint8[] interaction_control_mode
 # All 6 values in force and impedance parameter vectors have to be filled,
 # If a control mode is not used in a Cartesian dimension,
 # the corresponding parameters will be ignored.
+
+## Parameters for Constrained Zero-G Behaviors
+# Allow arbitrary rotational displacements from the current orientation for constrained zero-G
```

Comment:
Unless this has already been committed to Intera, can you modify the comment lines to be 80 characters or less?

Reply:
Oh.. This has already been checked into master.

Reply:
In general, I recommend putting up both reviews at the same time. In this case, it might be worth it to put in a second smaller commit into master.

Reply:
I see. Sure, I can do that. I will put up both reviews together next time.

Reply:
As changing these in the future will result in breaking ROS API (bc of incompatible md5 sums), can you make the modification both here and in Intera?